extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/TransactionManagerConfiguration.java

```diff
@@ -19,8 +16,92 @@
     public String nodeName;
 
     /**
-     * The default transaction timeout
+     * The default transaction timeout in seconds
```

Comment:
The type is `Duration`, so the unit is understood already to be seconds unless it is overridden.  To that end, since you're giving a default of 300 seconds, it's probably more user-friendly to use the string `5m` instead.

Reply:
Do you mean 

```java
    @ConfigItem(defaultValue = "5m")
    public Optional<Duration> defaultTransactionTimeout;
```

If so, then that would be liable to be misinterpreted. How would he define seconds or hour units for example.

It is less ambiguous if we pick a single unit (namely seconds) and say that any value the user specifies will always be interpreted as seconds.

Reply:
That's not how we configure time durations in Quarkus.

Seconds are the default unit for all duration properties.  Users can use an abbreviated version of ISO-8601 duration strings to specify other units.  For example, `10` or `10S` or `PT10S` all indicate 10 seconds; `10M` is 10 minutes; `5H10M2S` is 5 hours, 10 minutes, and 2 seconds; `P2DT4H` is 2 days, 4 hours, etc.

It's the same format used for all duration-typed properties in all of Quarkus, so I would not be terribly worried about users getting confused about the format.